### PR TITLE
Added NodeKit

### PR DESCRIPTION
**Pull request by mtgnoah**

### CODE REVIEW

File: data/ecosystems/a/avalanche.toml [hunk 1]
```diff
@@ -91,6 +91,7 @@ sub_ecosystems = [
   "Nereus Finance",
   "NFTrade",
   "Niftyx",
+  "NodeKit",
```
Comment (eherrerosj) [inline data/ecosystems/a/avalanche.toml:94]: I don't think we have an ecosystem called NodeKit, can you review please?

**Review by eherrerosj — CHANGES_REQUESTED** [review]

File: data/ecosystems/a/avalanche.toml (hunk 1 above)
Comment (mtgnoah) [reply data/ecosystems/a/avalanche.toml:94]: Hey so if I am building on the Avalanche ecosystem do I add a new ecosystem and also keep the information I put here?

Comment (eherrerosj) [reply data/ecosystems/a/avalanche.toml:94]: That is correct, you can create an ecosystem called say `nodekit.toml` under `data/ecosytems/n` and with `title = "NodeKit"`, and then you would keep the current line here

**Comment by eherrerosj** [comment]
@mtgnoah, any chance you'll be able to go through my suggestions within the next couple weeks? This way we can merge this PR and make the repo development count towards the 2023 Dev Report

**Comment by eherrerosj** [comment]
> Hey @eherrerosj, if this PR looks good, could we please add it? Looks like @mtgnoah pushed some changes

I'm seeing there's conflict happening in nodekit.toml @ashucoder9, @mtgnoah, could you fix it so I can merge?

**Comment by ashucoder9** [comment]
> > Hey @eherrerosj, if this PR looks good, could we please add it? Looks like @mtgnoah pushed some changes
> 
> I'm seeing there's conflict happening in nodekit.toml @ashucoder9, @mtgnoah, could you fix it so I can merge?

I guess only you and @mtgnoah have write access!

**Comment by mtgnoah** [comment]
> > Hey @eherrerosj, if this PR looks good, could we please add it? Looks like @mtgnoah pushed some changes
> 
> I'm seeing there's conflict happening in nodekit.toml @ashucoder9, @mtgnoah, could you fix it so I can merge?

I think I fixed it

**Review by eherrerosj — CHANGES_REQUESTED** [review]
small changed needed

**Comment by eherrerosj** [comment]
Hi @ashucoder9, @mtgnoah, we wanted to give you a heads-up that we are planning to freeze the data set that will be used for the analysis during the next few days. If you want your changes to be included and have an impact on this year’s report, please ensure that the requested modifications are made and merged prior to the data freeze.

Let's aim to have your PR finalized as soon as possible to make it into this window. If you have any questions or need further clarification, feel free to reply here or reach out directly.

**Comment by mtgnoah** [comment]
Hi @eherrerosj I think I have now made all the required changes.

**Review by eherrerosj — APPROVED** [review]
LGTM now. Thanks! Merging

**Pull request merged**